charts/prometheus-snmp-exporter/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v1
 description: Prometheus SNMP Exporter
 name: prometheus-snmp-exporter
-version: 1.8.1
+version: 1.8.2
```

Comment:
Because we're changing the default of one of the Chart values (`serviceMonitor.namespace` from `"monitoring"` to just `""`, i.e. empty) please can you change this to be a major version change, i.e. 2.0.0, and update the [README's Upgrading Chart](https://github.com/prometheus-community/helm-charts/tree/main/charts/prometheus-snmp-exporter#upgrading-chart) section to have a new section for upgrading to this new version which states that this value has changed, just so users aren't caught out. Thanks!

Reply:
All set.